tests/integration/ceph_base_object_test.go

```diff
@@ -65,17 +65,14 @@ var (
 // Create object store, Create User, Connect to Object Store, Create Bucket, Read/Write/Delete to bucket,
 // Check issues in MGRs, Delete Bucket and Delete user
 // Test for ObjectStore with and without TLS enabled
-func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
-	storeName := "tlsteststore"
-	logger.Info("Object Storage End To End Integration Test with TLS enabled - Create Object Store, User,Bucket and read/write to bucket")
-	logger.Infof("Running on Rook Cluster %s", namespace)
-	createCephObjectStore(s, helper, k8sh, namespace, storeName, 3, true)
-	testObjectStoreOperations(s, helper, k8sh, namespace, storeName)
+func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, namespace string, tlsEnable bool) {
+	storeName := "test-store"
+	if tlsEnable {
+		storeName = "tls-test-store"
+	}
 
-	storeName = "teststore"
-	logger.Info("Object Storage End To End Integration Test without TLS - Create Object Store, User,Bucket and read/write to bucket")
 	logger.Infof("Running on Rook Cluster %s", namespace)
-	createCephObjectStore(s, helper, k8sh, namespace, storeName, 3, false)
+	createCephObjectStore(s, helper, k8sh, namespace, storeName, 3, tlsEnable)
 	testObjectStoreOperations(s, helper, k8sh, namespace, storeName)
 }
```

Comment:
Allow for running TLS and non-TLS tests in separate subtests of the CephObjectSuite. This makes the tests visually separated in the final report

```
--- PASS: TestCephObjectSuite (311.37s)
    --- PASS: TestCephObjectSuite/TestWithTLS (92.33s)
        --- PASS: TestCephObjectSuite/TestWithTLS/create_CephObjectStore (35.86s)
        --- PASS: TestCephObjectSuite/TestWithTLS/create_CephObjectStoreUser_"tls-test-store" (6.33s)
        --- PASS: TestCephObjectSuite/TestWithTLS/verify_ceph_object_store_"tls-test-store"_status (0.00s)
        --- PASS: TestCephObjectSuite/TestWithTLS/create_ObjectBucketClaim_with_reclaim_policy_delete (3.44s)
        --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket (2.64s)
            --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/put_object_on_OBC_bucket (0.01s)
            --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/get_object_on_OBC_bucket (0.00s)
            --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/test_quota_enforcement_on_OBC_bucket (0.01s)
            --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/test_update_quota_on_OBC_bucket (2.44s)
            --- PASS: TestCephObjectSuite/TestWithTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/delete_objects_on_OBC_bucket (0.02s)
        --- PASS: TestCephObjectSuite/TestWithTLS/Regression_check:_Verify_bucket_does_not_revert_to_Pending_phase (15.35s)
        --- PASS: TestCephObjectSuite/TestWithTLS/delete_CephObjectStore_should_be_blocked_by_OBC_bucket_and_CephObjectStoreUser (2.15s)
        --- PASS: TestCephObjectSuite/TestWithTLS/delete_OBC (0.94s)
        --- PASS: TestCephObjectSuite/TestWithTLS/delete_CephObjectStoreUser (0.45s)
        --- PASS: TestCephObjectSuite/TestWithTLS/check_that_mgrs_are_not_in_a_crashloop (0.01s)
        --- PASS: TestCephObjectSuite/TestWithTLS/CephObjectStore_should_delete_now_that_dependents_are_gone (24.55s)
    --- PASS: TestCephObjectSuite/TestWithoutTLS (86.57s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/create_CephObjectStore (30.71s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/create_CephObjectStoreUser_"test-store" (6.12s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/verify_ceph_object_store_"test-store"_status (0.00s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/create_ObjectBucketClaim_with_reclaim_policy_delete (3.41s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket (2.91s)
            --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/put_object_on_OBC_bucket (0.01s)
            --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/get_object_on_OBC_bucket (0.00s)
            --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/test_quota_enforcement_on_OBC_bucket (0.01s)
            --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/test_update_quota_on_OBC_bucket (2.64s)
            --- PASS: TestCephObjectSuite/TestWithoutTLS/use_S3_client_to_put_and_get_objects_on_OBC_bucket/delete_objects_on_OBC_bucket (0.06s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/Regression_check:_Verify_bucket_does_not_revert_to_Pending_phase (15.33s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/delete_CephObjectStore_should_be_blocked_by_OBC_bucket_and_CephObjectStoreUser (2.12s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/delete_OBC (0.93s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/delete_CephObjectStoreUser (0.42s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/check_that_mgrs_are_not_in_a_crashloop (0.01s)
        --- PASS: TestCephObjectSuite/TestWithoutTLS/CephObjectStore_should_delete_now_that_dependents_are_gone (24.60s)
PASS
ok  	github.com/rook/rook/tests/integration	311.393s
```